tools/build_defs/framework.bzl

```diff
@@ -331,8 +333,7 @@ def wrap_outputs(ctx, lib_name, configure_name, script_text):
     cleanup_on_success_function = create_function(
         ctx,
         "cleanup_on_success",
-        """##echo## \"rules_foreign_cc: Cleaning temp directories\"
-rm -rf $BUILD_TMPDIR $EXT_BUILD_DEPS""",
+        """rm -rf $BUILD_TMPDIR $EXT_BUILD_DEPS""",
```

Comment:
@oquenchil would it be possible to at least cherry-pick this part of the pull request into master?

Then we can deal about the rest separately. (Although the rest of this PR would also be really nice to have, the leftover temp directories make things hard to debug.)